cmd/restic/cmd_prune.go

```diff
@@ -207,10 +207,12 @@ func pruneRepository(gopts GlobalOptions, repo restic.Repository) error {
 	}
 	bar.Done()
 
-	if len(usedBlobs) > stats.blobs {
-		return errors.Fatalf("number of used blobs is larger than number of available blobs!\n" +
-			"Please report this error (along with the output of the 'prune' run) at\n" +
-			"https://github.com/restic/restic/issues/new")
+	for h := range usedBlobs {
+		if _, ok := blobCount[h]; !ok {
+			return errors.Fatalf("At least one data blob seems to be missing, aborting prune to prevent further data loss!\n" +
```

Comment:
Is it possible and useful to be more specific here, e.g. including the (shortened) blob ID, to ease troubleshooting?

Reply:
I've added the ID of the first missing blob. It would also be possible to dump a possibly gigantic list with all missing blobs. Not sure whether that's a good idea.